pr_best_practices.py

```diff
@@ -5,10 +5,21 @@
 import requests
 
 
+def check_jira_issues_public(text):
+    for match in re.findall(r"[A-Z][A-Z0-9]+-[0-9]+", text):
+        url = f"https://issues.redhat.com/browse/{match}"
+        res = requests.get(url)
+
+        if res.status_code != 200:
+            print("⛔ Assumed issue {match!r} is not publicly accessible.")
```

Comment:
At this point, the check doesn't discern between issues that aren't accessible and non-existent issues. Not even sure though the Jira API returns different error codes.

Reply:
It doesn't :(